src/pages/players/[id]-[username].astro

```diff
@@ -51,9 +51,12 @@ const highestLeague = player?.leaderboard_entries?.reduce(
   (acc, entry) => (entry.points > acc.points ? entry : acc),
   player.leaderboard_entries[0]
 )
+const ogImageUrl= `/og-image/players/${playerId}-${player.nickname}.png`
```

Comment:
url should contain `player.last_match_ended_at` (can be query param) so we can efficiently cache it on CDN with automatic invalidation

Reply:
`/og-image/players/${playerId}-${player.nickname}.png?timestamp=12323490234` can be enough

Reply:
Added ?timestamp=${player.last_match_ended_at} but would Date.now() be better? Just saw that comment

Reply:
`Date.now()` would certainly not be better (as it would cause cache invalidation all the time)